### PR TITLE
Update API.md to reflect we are now on v5

### PR DESCRIPTION
**Pull request by bac**

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Charm store API
 
-The current live API lives at https://api.jujucharms.com/charmstore/v4
+The current live API lives at https://api.jujucharms.com/charmstore/v5
```
Comment (rogpeppe) [inline docs/API.md:3]: It's at v5-unstable

**Comment by rogpeppe** [comment]
LGTM except that the Go stuff lives in v5-unstable, not v5.

File: docs/API.md (hunk 1 above)
Comment (mhilton) [inline docs/API.md:3]: Ignore roger he's talking out of his elbow.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1694/
Test PASSed.

**Comment by mhilton** [comment]
:+1: You're URLs are fine, v5-unstable is only for package names.

File: docs/API.md (hunk 1 above)
Comment (bac) [inline docs/API.md:3]: I don't understand. The API is at v5 as in:

https://api.jujucharms.com/charmstore/v5/wordpress/meta/charm-metadata

right?

Comment (rogpeppe) [inline docs/API.md:3]: "Elbow" is a polite word for it. I'd've used something a little more pithy.

**Comment by bac** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Pull request merged**